docs/_includes/getnighthawk.html

```diff
@@ -1,11 +1,11 @@
 <div class="container-g">
     <h1 class="title-g">
         What is The Purpose Of <br/>
-        GetNighthawk
+        Nighthawk
     </h1>
     <br/>
     <p class="para-g">
-        GetNighthawk provides generally-available distributions of Nighthawk under different architectures and platforms and easy-to-use tooling for installation and operation. This will include creating distributions of Nighthawk as well as augmenting existing tooling, Meshery, to retrieve these arch-specific packages and update their deployments.
+        The generally-available distributions of Nighthawk is provided under different architectures and platforms and easy-to-use tooling for installation and operation. This will include creating distributions of Nighthawk as well as augmenting existing tooling, Meshery, to retrieve these arch-specific packages and update their deployments.
```

Comment:
```suggestion
        The generally-available distributions of Nighthawk are provided under different architectures and platforms and provide easy-to-use tooling for installation and operation. This will include creating distributions of Nighthawk as well as augmenting existing tooling, Meshery, to retrieve these arch-specific packages and to update their deployments.
```

Reply:
@Mr-DG-Wick Thank you for the review. I think we can make the last line even better : 
`
This will include creating distributions of Nighthawk and augmenting existing tooling, Meshery, to retrieve these arch-specific packages and update their deployments.
`

Do let me know what you think.

Reply:
@toth2000 your phrasing is much better. But now that I read the whole paragraph, the first sentence and the second sentence are not going that well together. The previous version of it also was a bit hard to digest for me. 
So, in summary I feel like there is a missing sentence between the first sentence and the second sentence.

Reply:
@Mr-DG-Wick yeah, I agree.

Reply:
@Mr-DG-Wick I worked some changes, hope it is more clear now.

Reply:
@toth2000 yup looks good!.